lib/libwebsockets.h

```diff
@@ -234,15 +234,7 @@ struct libwebsocket_pollargs {
     int prev_events;   // the previous event mask
 };
 
-#ifdef _WIN32
```

Comment:
You should not remove the normal Win32 support. All the MinGW support should have its own `#ifdef`s if they need something different